### PR TITLE
[libogg] Upgrade to 1.3.5

### PR DESCRIPTION
**Pull request by evpobr**
* Upgrade to 1.3.5.
* Convert CONTROL to vcpkg.json
* Change homepage to https://www.xiph.org/ogg (as in README.md)
* Download from the main repo https://gitlab.xiph.org/xiph/ogg

### CODE REVIEW

File: ports/libogg/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "libogg",
+  "version-string": "1.3.5",
```
Comment (JonLiu1993) [inline ports/libogg/vcpkg.json:3]: @evpobr ,We should use "version" instead of "version-string"

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
Could you please take a look:
[install-x64-uwp-dbg-out.log](https://github.com/microsoft/vcpkg/files/6684382/install-x64-uwp-dbg-out.log)

**Comment by evpobr** [comment]
> Could you please take a look:
> [install-x64-uwp-dbg-out.log](https://github.com/microsoft/vcpkg/files/6684382/install-x64-uwp-dbg-out.log)

I don't think it is related to libogg:

    4>D:\buildtrees\simage\src\mage-1.8.0-17f988a113.clean\src\simage_tiff.c(88,18): error C4996: 'uint16': libtiff type deprecated; please use corresponding C99 stdint.h type [D:\buildtrees\simage\x64-uwp-dbg\simage.vcxproj]

File: ports/libogg/vcpkg.json (hunk 1 above)
Comment (evpobr) [inline ports/libogg/vcpkg.json:3]:
```suggestion
  "version": "1.3.5",
```

**Comment by JonLiu1993** [comment]
> > Could you please take a look:
> > [install-x64-uwp-dbg-out.log](https://github.com/microsoft/vcpkg/files/6684382/install-x64-uwp-dbg-out.log)
> 
> I don't think it is related to libogg:
> 
> ```
> 4>D:\buildtrees\simage\src\mage-1.8.0-17f988a113.clean\src\simage_tiff.c(88,18): error C4996: 'uint16': libtiff type deprecated; please use corresponding C99 stdint.h type [D:\buildtrees\simage\x64-uwp-dbg\simage.vcxproj]
> ```

I will rerun CI

**Comment by JonLiu1993** [comment]
After change the Suggested  please run "./vcpkg x-add-version libogg --overwrite-version" and commit the change again

**Comment by dg0yt** [comment]
The simage/tiff issue is a bug in simage dependency setup affecting multiple PRs.

**Comment by evpobr** [comment]
> After change the Suggested please run "./vcpkg x-add-version libogg --overwrite-version" and commit the change again

I can do it late tonight.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by ras0219-msft** [comment]
Awesome, thanks for another great PR!

**Pull request merged**